lib/process-modules.js

```diff
@@ -25,25 +26,16 @@ function processTimeSeries(timeSeriesData) {
   return textUpdate;
 }
 
-function getTextUpdate(module) {
-  var textUpdate = [],
-    timeSeriesUpdate;
-  if (module['module-type'] === 'grouped_timeseries') {
-    _.each(module.data, function (timeSeries) {
-      textUpdate.push(timeSeries.label);
-      timeSeriesUpdate = processTimeSeries(timeSeries.values);
-      textUpdate = textUpdate.concat(timeSeriesUpdate);
-    });
-    return textUpdate;
-  } else {
-    return processTimeSeries(module.data);
-  }
+function getTextUpdate(moduleData) {
+  var formattedModule = new Delta(moduleData);
+  return processTimeSeries(formattedModule.data);
 }
 
 module.exports = function (modules) {
-
+  var dummyModule = new Module(),
```

Comment:
I assume you're gonna use the new prototype property for this so I'll wait for you to put in the updated version.

:eggplant: